truenas_install/__main__.py

```diff
@@ -184,9 +184,11 @@ def enable_user_services(root, old_root):
                     os.makedirs(f"{root}/boot/efi", exist_ok=True)
                     for disk in disks:
                         run_command(["chroot", root, "grub-install", "--target=i386-pc", f"/dev/{disk}"])
-
-                        run_command(["chroot", root, "mkdosfs", "-F", "32", "-s", "1", "-n", "EFI", f"/dev/{disk}2"])
-                        run_command(["chroot", root, "mount", "-t", "vfat", f"/dev/{disk}2", "/boot/efi"])
+                        # Check if nvme disk and instead use nvme naming convention.
+                        if "nvme" in disk: partition = "{disk}p2"
```

Comment:
The correct PEP8 syntax is:
```
if "nvme" in disk:
    partition = "{disk}p2"
else:
    partition = "{disk}2"
```

Reply:
Also you've forgot to make them f-strings e.g. `f"{disk}2"`

Reply:
Ah I just saw this after I committed more changes, I honestly never touched Python in my life so I'm winging it. I discovered the issue with the f-strings after it failed for me on my VM. I'll recommit with the PEP8 syntax now.